scripts/add_field_for_stuck_registrations/add_field_to_stuck_registration_contacts.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+import asyncio
+import csv
+import sys
+
+from ndoh_hub.utils import rapidpro
+
+
+def process(contact_id):
+    contact = rapidpro.get_contacts(uuid=contact_id).first()
+    params = {"stuck_in_consent_state": "True"}
+    rapidpro.update_contact(contact, fields=params)
```

Comment:
The first argument to `update_contact` is the UUID or the URN of the contact to update: https://github.com/rapidpro/rapidpro-python/blob/master/temba_client/v2/__init__.py#L461-L476 .

So we don't need to first do a `get_contacts`, we can just call `update_contact` with the UUID.

Reply:
okay